src/Interpreters/Context.h

```diff
@@ -372,6 +372,7 @@ class ContextData
             partitions = rhs.partitions;
             projections = rhs.projections;
             views = rhs.views;
+            skip_indexes = rhs.skip_indexes;
```

Comment:
Can we somehow maintain the relative order, i.e. "skip_indexes" is the next line after "projections".